pkg/node/sendoperation/sendoperation.go

```diff
@@ -177,30 +177,30 @@ func message(expiry uint64, fee uint64, operation Operation) []byte {
 	return msg
 }
 
-func DecodeMessage64(msgB64 string) ([]byte, error) {
+func DecodeMessage64(msgB64 string) ([]byte, uint64, uint64, error) {
```

Comment:
could you add a function comment godoc here to explain what are the returned values?